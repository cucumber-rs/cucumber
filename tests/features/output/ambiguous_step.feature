Feature: ambiguous
  Scenario: ambiguous
    Given foo is 0 ambiguous
