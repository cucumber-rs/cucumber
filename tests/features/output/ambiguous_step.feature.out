Started
Feature(Feature { keyword: "Feature", name: "ambiguous", description: None, background: None, scenarios: [Scenario { keyword: "Scenario", name: "ambiguous", steps: [Step { keyword: "Given", ty: Given, value: "foo is 0 ambiguous", docstring: None, table: None, position: LineCol { line: 3, col: 5 } }], examples: [], tags: [], position: LineCol { line: 2, col: 3 } }], rules: [], tags: [], position: LineCol { line: 1, col: 1 }, }, Started)
Feature(Feature { keyword: "Feature", name: "ambiguous", description: None, background: None, scenarios: [Scenario { keyword: "Scenario", name: "ambiguous", steps: [Step { keyword: "Given", ty: Given, value: "foo is 0 ambiguous", docstring: None, table: None, position: LineCol { line: 3, col: 5 } }], examples: [], tags: [], position: LineCol { line: 2, col: 3 } }], rules: [], tags: [], position: LineCol { line: 1, col: 1 }, }, Scenario(Scenario { keyword: "Scenario", name: "ambiguous", steps: [Step { keyword: "Given", ty: Given, value: "foo is 0 ambiguous", docstring: None, table: None, position: LineCol { line: 3, col: 5 } }], examples: [], tags: [], position: LineCol { line: 2, col: 3 } }, Started))
Feature(Feature { keyword: "Feature", name: "ambiguous", description: None, background: None, scenarios: [Scenario { keyword: "Scenario", name: "ambiguous", steps: [Step { keyword: "Given", ty: Given, value: "foo is 0 ambiguous", docstring: None, table: None, position: LineCol { line: 3, col: 5 } }], examples: [], tags: [], position: LineCol { line: 2, col: 3 } }], rules: [], tags: [], position: LineCol { line: 1, col: 1 }, }, Scenario(Scenario { keyword: "Scenario", name: "ambiguous", steps: [Step { keyword: "Given", ty: Given, value: "foo is 0 ambiguous", docstring: None, table: None, position: LineCol { line: 3, col: 5 } }], examples: [], tags: [], position: LineCol { line: 2, col: 3 } }, Step(Step { keyword: "Given", ty: Given, value: "foo is 0 ambiguous", docstring: None, table: None, position: LineCol { line: 3, col: 5 } }, Started)))
Feature(Feature { keyword: "Feature", name: "ambiguous", description: None, background: None, scenarios: [Scenario { keyword: "Scenario", name: "ambiguous", steps: [Step { keyword: "Given", ty: Given, value: "foo is 0 ambiguous", docstring: None, table: None, position: LineCol { line: 3, col: 5 } }], examples: [], tags: [], position: LineCol { line: 2, col: 3 } }], rules: [], tags: [], position: LineCol { line: 1, col: 1 }, }, Scenario(Scenario { keyword: "Scenario", name: "ambiguous", steps: [Step { keyword: "Given", ty: Given, value: "foo is 0 ambiguous", docstring: None, table: None, position: LineCol { line: 3, col: 5 } }], examples: [], tags: [], position: LineCol { line: 2, col: 3 } }, Step(Step { keyword: "Given", ty: Given, value: "foo is 0 ambiguous", docstring: None, table: None, position: LineCol { line: 3, col: 5 } }, Failed(None, None, AmbiguousMatch(AmbiguousMatchError { possible_matches: [(HashableRegex(foo is (\d+)), Some(Location { line: 14, column: 1 })), (HashableRegex(foo is (\d+) ambiguous), Some(Location { line: 22, column: 1 }))] })))))
Feature(Feature { keyword: "Feature", name: "ambiguous", description: None, background: None, scenarios: [Scenario { keyword: "Scenario", name: "ambiguous", steps: [Step { keyword: "Given", ty: Given, value: "foo is 0 ambiguous", docstring: None, table: None, position: LineCol { line: 3, col: 5 } }], examples: [], tags: [], position: LineCol { line: 2, col: 3 } }], rules: [], tags: [], position: LineCol { line: 1, col: 1 }, }, Scenario(Scenario { keyword: "Scenario", name: "ambiguous", steps: [Step { keyword: "Given", ty: Given, value: "foo is 0 ambiguous", docstring: None, table: None, position: LineCol { line: 3, col: 5 } }], examples: [], tags: [], position: LineCol { line: 2, col: 3 } }, Finished))
Feature(Feature { keyword: "Feature", name: "ambiguous", description: None, background: None, scenarios: [Scenario { keyword: "Scenario", name: "ambiguous", steps: [Step { keyword: "Given", ty: Given, value: "foo is 0 ambiguous", docstring: None, table: None, position: LineCol { line: 3, col: 5 } }], examples: [], tags: [], position: LineCol { line: 2, col: 3 } }], rules: [], tags: [], position: LineCol { line: 1, col: 1 }, }, Finished)
Finished
